solutions/cses-1625.mdx

```diff
@@ -0,0 +1,343 @@
+---
+id: cses-1625
+source: CSES (TODO -- convert to something like `USACO Silver 2017 January`)
+title: Grid Paths
+author: Vivian Han
+---
+
+## Explanation
+
+Adapted from https://usaco.guide/CPH.pdf#page=61 
+
+*Please note: the running times and number of recursive calls listed are for a separate but very similar problem (#paths from upper left to *lower right* corner on a 7x7 grid); they may not be entirely accurate, but they illustrate the effect of each optimization. The numbers are also C++ specific, so Java and Python users will experience significantly different runtimes, which will be addressed later in the explanation.*
```

Comment:
wrap in info block or warning block? (whatever you think is more appropriate) https://usaco.guide/general/working-mdx#info-block